webcompat/form.py

```diff
@@ -236,9 +247,10 @@ def build_formdata(form_object):
         summary = u'{0} - {1}'.format(normalized_url, problem_summary)
 
     metadata_keys = ['browser', 'ua_header', 'reported_with']
-    extra_label = form_object.get('extra_label', None)
-    if extra_label in app.config['EXTRA_LABELS']:
-        metadata_keys.append('extra_label')
+    extra_labels = form_object.get('extra_labels', None)
+    if extra_labels:
+        # Note: we check if the label is allowed in normalize_metadata
```

Comment:
```python
    if extra_labels:
        extra_labels = [normalize_metadata(label)
                        for label in extra_labels
                        if label in app.config['EXTRA_LABELS']]
        metadata_keys.append('extra_labels')
```

This way we add only two instructions instead of all the juggling in normalize_metadata

Reply:
This is nicer. We don't do anything with that list you created... but I think we can create a separate `extra_labels` and merge with the `metadata` list before joining as a string inside `get_metadata`.

Reply:
OK, I can remove the `# Note` comment. I took your suggestion, but moved it here:

https://github.com/webcompat/webcompat.com/pull/2007/files#diff-f1975977b06efb529b8c60f23a7bc4eeR131